src/main/java/de/neuland/jade4j/util/ArgumentSplitter.java

```diff
@@ -0,0 +1,54 @@
+package de.neuland.jade4j.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author dusan.zatkovsky, 2/5/15
+ */
+public class ArgumentSplitter {
+
+    private static final char argumentDelimiter = ',';
+    private final String arguments;
+    private List<String> argList = new ArrayList<String>();
+    private StringBuffer sb;
+
+    public static List<String> split(String arguments) {
+        return new ArgumentSplitter(arguments).splitArguments();
+    }
+
+    public ArgumentSplitter(String arguments) {
```

Comment:
There is no point exposing this constructor if there is not even a single public method.